src/Auth/AbstractPasswordHasher.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Thats more than silly :/ declare() should be on the same line, by common sense.

Reply:
Common sense and PSR standards don't always go hand in hand.